packages/node-resolve/src/resolveImportSpecifiers.js

```diff
@@ -0,0 +1,197 @@
+import fs from 'fs';
+import path from 'path';
+import { promisify } from 'util';
+
+import resolve from 'resolve';
+
+import { getPackageName } from './util';
+import { exists, realpath } from './fs';
+
+const resolveImportPath = promisify(resolve);
+const readFile = promisify(fs.readFile);
+
+const pathNotFoundError = (subPath, pkgPath) =>
+  new Error(`Package subpath '${subPath}' is not defined by "exports" in ${pkgPath}`);
+
+function findExportKeyMatch(exportMap, subPath) {
+  for (const [key, value] of Object.entries(exportMap)) {
+    if (key.endsWith('*') && value.includes('*')) {
```

Comment:
```suggestion
    if (key.endsWith('*')) {
```

`value` doesn't strictly have to include `*` - it is possible to have a static map theoretically.